config/options.go

```diff
@@ -81,7 +81,7 @@ func defaultResolver(input map[string]interface{}) error {
 		} else if len(args) > 1 { // default value
 			return args[1]
 		}
-		return ""
+		return args[0]
```

Comment:
If `resolver` is added to each watch loop, it's OK to return empty `""` for not found value, because env vars are not support to be added by `watch` now, and new added palceholder will not be resolved again.

Reply:
got it! I misunderstand your last comment haha